assets/certificateTemplate.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>[TEST] Corona School Bescheinigung</title>
+</head>
+<body>
+    Name des Studenten: %NAMESTUDENT%<br/>
+    Name des Schülers: %NAMESCHUELER%<br/>
+    Heutiges Datum: %DATUMHEUTE%<br/>
+    Startdatum: %SCHUELERSTART%<br/>
+    Enddatum: %SCHUELERENDE%<br/>
+    Fächer: %SCHUELERFAECHER%<br/>
+    Text Schüler: %SCHUELERFREITEXT%<br/>
+    Stunden pro Woche: %SCHUELERPROWOCHE%<br/>
+    Stunden gesamt: %SCHUELERGESAMT%<br/>
+    Mium: %MEDIUM%<br/>
```

Comment:
Probably "Medium", but doesn't really matter 😂

Reply:
To not accidentially commit the live version I used `git update-index --assume-unchanged` ... Actually fixed that typo, just didn't appear in the diff ... done now :)